tools/oegenerate/host/evidence.cpp

```diff
@@ -230,10 +230,12 @@ void dump_claims(const oe_claim_t* claims, size_t claims_length)
 {
     printf("\n%zu OE claims retrieved:\n\n", claims_length);
     size_t i = 0;
+    // id version, security version, attributes(bit flags)
     for (; i < 3; i++)
     {
         printf("claims[%zu]: %s\n%u\n\n", i, claims[i].name, *claims[i].value);
     }
+    // unique id, signer id, product id, format uuid, sgx claims
```

Comment:
Seems odd to call out "sgx" in particular in this comment since the tool name, the function name, and all the code below appear to all be TEE-agnostic.
```suggestion
    // unique id, signer id, product id, format uuid, TEE-specific claims
```